src/app/features/group-list/group-item/group-item.component.scss

```diff
@@ -0,0 +1,28 @@
+.group-item-container {
+    padding: 8px 16px;
+    position: relative;
+}
+
+.group-item {
+    border: 1px solid black;
+    border-radius: 20px;
+    background-color: #F5F5F5;
+}
+
+.group-item-container button {
+    position: absolute;
+    top: 0px;
+    right: 0px;
+}
```

Comment:
tutaj można by było dodać odrobinę marginów ponieważ ten plus jest strasznie przyklejony do lewej krawędzi ekranu. Ja wiem że to jest zrobione zgodnie z figmą ale chciałem to wypunktować.

Reply:
Dodałem margines, aby przycisk nie był przyklejony do krawędzi.